WalletWasabi.Fluent/Views/Settings/AdvancedSettingsTabView.axaml

```diff
@@ -0,0 +1,18 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:settings="using:WalletWasabi.Fluent.ViewModels.Settings"
+             mc:Ignorable="d" d:DesignWidth="428" d:DesignHeight="371"
+             x:Class="WalletWasabi.Fluent.Views.Settings.AdvancedSettingsTabView"
+             x:CompileBindings="True"
+             x:DataType="settings:AdvancedSettingsTabViewModel">
+  <StackPanel Classes="settingsLayout">
+
+    <DockPanel>
```

Comment:
Can you add a tooltip with a short description of what is it for?

Reply:
Sure will add

Reply:
Added